src/objectUtils.js

```diff
@@ -24,3 +24,22 @@ export function assignDeep(target, source) {
 	}
 	return output;
 }
+
+function alphabetizeKeys(target, blacklist = []) {
+	const usableKeys = [];
+	const keys = Object.keys(target);
+	for (let i = 0; i < keys.length; i++) {
+		if (blacklist.indexOf(keys[i]) > -1) {
```

Comment:
This could be clarified with an Array.includes(), like so:

`if (blacklist.includes(keys[i])) {`